src/services/analysis.ts

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios'
+
+// Utils
+import { delay, getRequestHeaders } from '../utils'
+
+export const analyze = async (data: {}, accessToken: string, url: string) => {
+  const headers = getRequestHeaders(accessToken)
+
+  try {
+    const response = await axios.post(url, { data }, { headers })
+
+    const uuidUrl = `${url}/${response.data.uuid}`
+    let status = response.data.status
+
+    // Initial status check
+    if (status === 'Finished') {
+      return getIssues(accessToken, `${uuidUrl}/issues`)
+    }
+
+    for (let i = 0; i < 10; i++) {
```

Comment:
Is 10 an arbitrary number or is there a specific reason not to run this loop more than that?

Reply:
after reading the doc more in deep I guess this is not to overload the API?

Reply:
Yeah, this isn't quite best way to handle it but `armlet` seems to be doing much more complicated stuff and I didn't want to import all the complexity into `mythxjs`. This is just an MVP that's in place as the issue this PR is trying to resolve describes